app/assets/stylesheets/app/order_management.scss

```diff
@@ -28,6 +28,10 @@
   top: 6px;
 }
 
+#js--reportAnIssueModal {
```

Comment:
More styles are needed as modal still looks off. Also, I think it's missing a Cancel button

<img width="603" alt="image" src="https://github.com/tablexi/nucore-open/assets/28798610/1fe9db37-da0f-4708-bb73-6f10a7d6353c">

Reply:
we have existing examples on the code, or you can look at bootstrap 2 docs